export const throwError = () => {
  throw new Error('Mock error')
}
